yocto-update-strategies.md

```diff
@@ -0,0 +1,651 @@
+class: center, middle, intro
+
+# Updates of Yocto-Based Projects: Effective strategies for ongoing maintenance
+
+### Yocto DevDay 2023
+
+## Tomasz Żyjewski
+
+<img src="remark-templates/3mdeb-presentation-template/images/logo.png" width="150px" style="margin-left:-20px">
+
+---
+# `whoami`
+
+.center[<img src="remark-templates/3mdeb-presentation-template/images/tomasz_zyjewski.png" width="150px">]
+
+.center[Tomasz Żyjewski]
+.center[_Embedded Systems Team Leader_]
+.right-column50[
+- over 4 years in 3mdeb
+- integration of update systems and OS creation for embedded devices
+- system security
+]
+.left-column50[
+- <a href="https://twitter.com/tomzy_0"><img src="remark-templates/3mdeb-presentation-template/images/twitter.png" width="24px" style="margin-bottom:-5px; margin-left:-15px"/> @tomzy_0</a>
+- <a href="mailto:tomasz.zyjewski@3mdeb.com"><img src="remark-templates/3mdeb-presentation-template/images/email.png" width="24px" style="margin-bottom:-5px; margin-left:-15px"/> tomasz.zyjewski@3mdeb.com</a>
+]
+
+
+---
+# Who we are ?
+
+.center[.image-15[![](remark-templates/3mdeb-presentation-template/images/coreboot-1024x1024.png)] .image-15[![](remark-templates/3mdeb-presentation-template/images/uefi-1024x1024.png)] .image-15[![](remark-templates/3mdeb-presentation-template/images/lvfs.png)] .image-15[![](remark-templates/3mdeb-presentation-template/images/yocto.png)]]
+.center[.image-35[![](remark-templates/3mdeb-presentation-template/images/openpower.svg)]]
+
+* coreboot licensed service providers since 2016 and leadership participants
+* UEFI Adopters since 2018
+* Yocto Participants and Embedded Linux experts since 2019
+* Official consultants for Linux Foundation fwupd/LVFS project since 2020
+* IBM OpenPOWER Foundation members since 2020
+
+---
+# Agenda
+
+* Project lifecycle
+* Project components
+* Update strategy
+* Performing update
+  - build environment
+  - external layers update
+  - custom layer update
+* Build troubleshooting
+  - bootloader
+  - kernel
+  - default applications
+  - custom applications
+* Veryfing update
+* Summary
+
+---
+# Project lifecycle
+
+* Initiation
```

Comment:
It does not fit on the slide perfectly.
Also, we have lots of text in this presentation. This slide is perfect for making image out of it.

Reply:
added img instead of text https://github.com/3mdeb/conferences/pull/7/commits/c5880dcd8abc43cf78a29114a79cfb0bbd2354f8